[2024-05-29T01:00:09.751+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T01:00:09.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dados_climaticos_estruturados.criar_pasta scheduled__2024-05-23T06:55:00+00:00 [queued]>
[2024-05-29T01:00:09.824+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dados_climaticos_estruturados.criar_pasta scheduled__2024-05-23T06:55:00+00:00 [queued]>
[2024-05-29T01:00:09.826+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-29T01:00:09.858+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): criar_pasta> on 2024-05-23 06:55:00+00:00
[2024-05-29T01:00:09.870+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=9474) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T01:00:09.873+0000] {standard_task_runner.py:63} INFO - Started process 9500 to run task
[2024-05-29T01:00:09.872+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dados_climaticos_estruturados', 'criar_pasta', 'scheduled__2024-05-23T06:55:00+00:00', '--job-id', '3109', '--raw', '--subdir', 'DAGS_FOLDER/weather/custom_dags/dados_climaticos_estruturados.py', '--cfg-path', '/tmp/tmpin36acrh']
[2024-05-29T01:00:09.875+0000] {standard_task_runner.py:91} INFO - Job 3109: Subtask criar_pasta
[2024-05-29T01:00:09.981+0000] {task_command.py:426} INFO - Running <TaskInstance: dados_climaticos_estruturados.criar_pasta scheduled__2024-05-23T06:55:00+00:00 [running]> on host 0473b835f7f4
[2024-05-29T01:00:10.239+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dados_climaticos_estruturados' AIRFLOW_CTX_TASK_ID='criar_pasta' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T06:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-23T06:55:00+00:00'
[2024-05-29T01:00:10.242+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T01:00:10.246+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T01:00:10.247+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T01:00:10.270+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dados_climaticos_estruturados, task_id=criar_pasta, run_id=scheduled__2024-05-23T06:55:00+00:00, execution_date=20240523T065500, start_date=20240529T010009, end_date=20240529T010010
[2024-05-29T01:00:10.346+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T01:00:10.389+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-29T01:00:10.392+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
